### PR TITLE
Update authorized_keys

### PR DESCRIPTION
**Pull request by MSevey**
# PULL REQUEST

## Overview
Add's davids new pubkey. 

## Checklist

- [ ] All git commits are signed. (REQUIRED)
- [ ] All new methods or updated methods have clear docstrings.
- [ ] Testing added or updated for new methods.
- [ ] Verified if any changes impact the WebPortal Health Checks.
- [ ] Appropriate documentation updated.
- [ ] Changelog file created.

### CODE REVIEW

**Review by meeh0w — APPROVED** [review]
Don't we want a proper comment on this key? `user@dev` may feel ambiguous in a few months.

File: setup-scripts/support/authorized_keys [hunk 1]
```diff
@@ -9,3 +9,4 @@ ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIN6Kcx8yetova4/ALUQHigo/PBMJO33ZTKOsg2jxSO2a
 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIDcenWnMQ6q/OEC4ZmQgjLDV2obWlR3fENV0zRGFvJF+ marcins@siasky.net
 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIB7prtVOTwtcSN9HkXum107RwcW5H8Vggx6Qv7T57ItT daniel@siasky.net
 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIH+4IrfEM9H16jqvPZncHkWWoHO4/BVq7d4pEyzK4e0W michal.leszczyk@skynetlabs.com
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDY48P1s8NAgjKFRN2/iUYXWbYsEb10kyPRPxlawtI4JqEMfDxan30c99ANe6badh3KS+EDLdviijD7FDKGP2sPSBWv4L0N996u5t8+tK+eon3kRaJfGfyHlAuQWoRIHU1IK/liMqcJ5RgpZxavwMYbz0nza3LVRgkzfAR4qtWOBL3TfWclRHqYC6+qvP9Y3mc/FJxXPCyFBukZt4z3kFSW4WzoZuPxubw7wfOhgiG+2NSjLMxmKbFOCqcDfD/UyAY/z3JY9VSOBqX2+XqTex4O0WYNt4GaVd3Spo5sWV2s+xAPnBLEr4+pN45reLs2CaJjjqygPWDPJ8fDUjK1qSgAmfOFiP79jDDsnU/+fL0HsnmfmVC79td5gwi7TTmoAGSXzCNKCgcW5Q1GecSdG2doXGuNYfcvX5S737K+9KU+ril5rUKxvNjHWugYStbgDU0xvKtI5buI+CYD+p2OP0qRupb9jmwuZ+EsIwSX09LzITjs0H9sZ3Jd36eTT9EmHPc= user@dev
```
Comment (kwypchlo) [inline setup-scripts/support/authorized_keys:12]: Didn't we want everyone to switch to ssh-ed25519 ?

https://levelup.gitconnected.com/demystifying-ssh-rsa-in-openssh-deprecation-notice-22feb1b52acd#:~:text=OpenSSH%20will%20be%20deprecating%20the,less%20than%20USD%2450K.

**Comment by kwypchlo** [comment]
> Don't we want a proper comment on this key? `user@dev` may feel ambiguous in a few months.

Yeah that should definitely be at least david@skynetlabs.com

File: setup-scripts/support/authorized_keys (hunk 1 above)
Comment (kwypchlo) [reply setup-scripts/support/authorized_keys:12]:
```suggestion
ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIHptEpqs57lhnHkfa+0SQgXQ4A63/YGV2cNTcGMQW+Jt david@skynetlabs.com
```

**Review by DavidVorick — APPROVED** [review]

**Pull request merged**